crates/ruff/src/rules/flake8_todos/rules/todos.rs

```diff
@@ -356,10 +356,10 @@ fn static_errors(
             if let Some(end_index) = trimmed.find(')') {
                 TextSize::try_from(end_index + 1).unwrap()
             } else {
-                trimmed.text_len()
+                TextSize::try_from(trimmed.find(' ') + 1).unwrap()
```

Comment:
I think this one is actually fine, we don't need to change this line. The author block should be treated as everything between parentheses - if there's no parenthesis, we can't assume that the author is a single word.